contribs/gmf/src/controllers/AbstractDesktopController.js

```diff
@@ -5,11 +5,13 @@ goog.require('gmf.controllers.AbstractAppController');
 goog.require('gmf.contextualdata.module');
 goog.require('gmf.drawing.module');
 goog.require('gmf.editing.module');
+goog.require('gmf.import.module'); // only for desktop_alt
 goog.require('gmf.permalink.shareComponent');
 goog.require('gmf.print.component');
 goog.require('gmf.profile.module');
 goog.require('gmf.raster.component');
 goog.require('ngeo.draw.features');
+goog.require('ngeo.googlestreetview.module');
```

Comment:
same?

Reply:
Previously, both was importing googlestreetview but perhaps that was an error.

Reply:
As I remember streetview is only in the desktop alt, than he should be imported only in desktop alt :-)